docs/references/express/require-auth.mdx

```diff
@@ -0,0 +1,33 @@
+---
+description: Learn how to protect routes in your Express.js application using the `requireAuth` middleware function.
+---
+
+# `requireAuth`
+
+`requireAuth` is a middleware function that you can use to protect routes in your Express.js application. This function checks if the user is authenticated or authorized, and returns a 401 status code if they are not.
```

Comment:
How would it check if they are authorized if it doesn't take any parameters around permissions?